src/sas/qtgui/Perspectives/Fitting/FittingWidget.py

```diff
@@ -1061,11 +1061,19 @@ def selectedParameters(self):
         return [s.row() for s in self.lstParams.selectionModel().selectedRows()
                 if self.isCheckable(s.row())]
 
-    def setParameterSelection(self, status=QtCore.Qt.Unchecked):
+    def setParameterSelection(self, status=QtCore.Qt.Unchecked, item=None):
         """
         Selected parameters are chosen for fitting
         """
         # Convert to proper indices and set requested enablement
+        if item is None:
```

Comment:
This breaks the '(De-)Select parameters for fitting' options when right-clicking on parameters. The `selectParameters()` and `deselectParameters()` methods will need to pass `item` somehow.

Reply:
Great catch! Fixed.